plugins/login-impl/src/index.ts

```diff
@@ -77,6 +77,8 @@ export default async (): Promise<LoginService> => {
       }
       if (result.result !== undefined) {
         setLoginInfo(result.result)
+        const token = result.result.token
+        await authorizeFileServer(token, 'https://localhost:18082')
```

Comment:
You need to move this code from login-impl, it should be in `main.ts` for products in case of authentication is ok. And listen for authentication from `main.ts` in case it is not. So login should not know about attachments.